cmd/alert.go

```diff
@@ -40,14 +51,21 @@ inactive = 0`,
 	 \_[CRITICAL] [PrometheusAlertmanagerJobMissing] - Job: [alertmanager] is firing - value: 1.00
 	 | total=2 firing=1 pending=0 inactive=1`,
 	Run: func(_ *cobra.Command, _ []string) {
+		// Convert --no-alerts-state to integer and validate input
+		noAlertsStateInt, err := convertStateToInt(cliAlertConfig.NoAlertsState)
```

Comment:
I would rename `noAlertsStateInt` to `noAlertsState`.

Reply:
Makes sense. Done